server/app/services/UserService.ts

```diff
@@ -64,7 +67,63 @@ export class UserService {
     if (!updatedUser) {
       throw new NotFoundError(`User with username ${username} was not found`);
     }
-
     return UserEntity.buildUser(updatedUser);
   }
+
+  public async forgotPassword(body: IUser): Promise<string> {
```

Comment:
this method should also import the Email Service to send the email too. THis way the /system/email can still be a private API

Reply:
its implemented, frontend /email call is removed

Reply:
as well the /email from NON_AUTH

Reply:
you didnt push yet

Reply:
my bad, its pushed